src/table.rs

```diff
@@ -170,6 +170,8 @@ generate_state_changes!(state_changes, {
         0x08..=0x17 => (Anywhere, Ignore),
         0x19        => (Anywhere, Ignore),
         0x1c..=0x1f => (Anywhere, Ignore),
-        0x20..=0xff => (Anywhere, OscPut),
+        0x20..=0x9b => (Anywhere, OscPut),
+        0x9c        => (Ground, None),
```

Comment:
Could you explain the idea behind `0x9c        => (Ground, None)`?

Reply:
0x9c is the C1 escape for the `ST` terminator.